fera_full.py

```
@@ -1,3 +1,189 @@
+                            tupleinfo = indexador_fera.addrels('relatorio', pathpdfinput = notindexed, pathdbext=sys.argv[1], \
+                                                                   rootx=global_settings.root, sqliteconnx=sqliteconn)
+                    except sqlite3.OperationalError as ex:  
+                        notindexed.append(pathp)
+                        try:
+                            sqliteconn.close()
+                        except:
+                            None
+                        indexador_fera.createNewDbFile(global_settings.root) 
+                        sqliteconn = utilities_general.connectDB(str(global_settings.pathdb))                           
+                        tupleinfo = indexador_fera.addrels('relatorio', pathpdfinput = notindexed, pathdbext=sys.argv[1], rootx=global_settings.root, sqliteconnx=sqliteconn)                            
+                        indexing = True
```

Ruff

Local variable `indexing` is assigned to but never used

Remove assignment to unused variable `indexing`

(F841)